### PR TITLE
Added entry to describe the getFullYear method in javascript

### PR DESCRIPTION
**Pull request by cguttweb**
### Description

I have added a new file called `getFullYear.md` which describes the `getFullYear()` method in JavaScript.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+Title: 'getFullYear()' 
+Description: 'Return the current full year'
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:3]:
```suggestion
Description: 'Return the current full year.' 
```
This is wrong, It gets the four digit year of any given date.

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Subjects: 
+  - 'Web Development'
+Tags: 
+  - 'Date'
+  - 'Method'
+CatalogContent: 
+  - 'paths/web-development'
+---
+
+The **getFullYear** method returns the current four digit year
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:13]:
```suggestion
The **getFullYear** method returns the current four digit year.
```
This is wrong, It gets the four digit year of any given date.

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Syntax
+
+```js
+myDate.getFullYear()
+```
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:19]: Needs a sentence describing the syntax, basically what `myDate` represents.

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## Example
+
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:22]: Needs a sentence explaining what the example is doing.

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+```js
+const getYear = new Date().getFullYear()
+// returns 2023
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:25]:
```suggestion
```
This entry will remain up beyond the current year, so remove this comment. Also, add a line that does something with the result, such as print it to the console.

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+## Syntax
+
+```js
+myDate.getFullYear()
+```
+
+## Example
+
+```js
+const getYear = new Date().getFullYear()
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:24]:
```suggestion
const getYear = new Date().getFullYear()
```
Change the variable name so it isn't the name of another JavaScript date method.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cguttweb I did a review and left a few comments.

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md (hunk 6 above)
Comment (cguttweb) [reply content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:24]: Good point I have changed this

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md (hunk 4 above)
Comment (cguttweb) [reply content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:22]: I have added this

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+The **getFullYear** method returns a four digit year for the date according to local time.
+
+## Syntax
+
+```js
+myDate.getFullYear()
+```
+
+The returned value of `.getFullYear()` method is an integer for the provided date.
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:21]:
```suggestion
The returned value of `.getFullYear()` method is an integer representing the year for the provided date, `myDate`.
```

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+## Example
+
+In the example below the `year` variable store the value of a new Date object and then logs the year to the console.
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:25]:
```suggestion
In the example below the `year` variable stores the year returned from of a new Date object and then logs it to the console.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cguttweb I did another review and left some edits.

**Review by SSwiniarski — APPROVED** [review]
@cguttweb Looks good for a second review.

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
+---
+Title: 'getFullYear()' 
+Description: 'Returns the four digit year for the given date'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:3]:
```suggestion
Description: 'Returns the four digit year for the given date.' 
```

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
+---
+Title: 'getFullYear()' 
+Description: 'Returns the four digit year for the given date' 
+Subjects: 
+  - 'Web Development'
+Tags: 
+  - 'Date'
+  - 'Method'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:8]:
```suggestion
  - 'Methods'
```

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+CatalogContent: 
+  - 'paths/web-development'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:10]:
```suggestion
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
```

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+The **getFullYear** method returns a four digit year for the date according to local time.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:13]:
```suggestion
The **`getFullYear()`** method returns a four digit year for the date according to local time.
```

File: content/javascript/concepts/dates/terms/getFullYear/getFullYear.md [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+
+## Syntax
+
+```js
+myDate.getFullYear()
+```
+
+The returned value of `.getFullYear()` method is an integer representing the year for the provided date `myDate`
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getFullYear/getFullYear.md:21]:
```suggestion
The returned value of `.getFullYear()` method is an integer representing the year for the provided date `myDate`.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cguttweb I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by cguttweb** [comment]
Hi @caupolicandiaz I've made the updates suggested please let me know if anything further. Thanks.

**Review by caupolicandiaz — APPROVED** [review]
@cguttweb I made a couple of small edits, including switching the example to a codebyte so that the output will remain current. I think this entry is all set.

**Comment by SSwiniarski** [comment]
@cguttweb Congrats on the new entry. Live link here :https://www.codecademy.com/resources/docs/javascript/dates/getFullYear

image

**Pull request merged**